helm/afm/templates/deployment.yaml

```diff
@@ -37,6 +37,11 @@ spec:
             - name: grpc
               containerPort: 8080
               protocol: TCP
+          readinessProbe:
+            failureThreshold: 3
```

Comment:
Please consider adding these values to values.yaml as in [fybrik](https://github.com/fybrik/fybrik/blob/master/charts/fybrik/values.yaml#L85)

Reply:
The main value in readinessProbe is the port that is defined here and not in the values.